CMakeLists.txt

```diff
@@ -197,8 +197,8 @@ endif(${MSVC12})
 # Common flags for MSVC
 if(${MSVC})
   message("Detected compiler as MSVC")
-  SET (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /W1 /bigobj /EHsc -DBOOST_ALL_NO_LIB -D_SCL_SECURE_NO_WARNINGS")
-  SET (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W1 /bigobj /EHsc -DBOOST_ALL_NO_LIB -D_SCL_SECURE_NO_WARNINGS")
+  SET (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /W1 /bigobj /EHsc -DBOOST_ALL_NO_LIB -D_SCL_SECURE_NO_WARNINGS -DNOMINMAX")
```

Comment:
@roigcarlo check this

Reply:
This should no longer be necessary, we solved it globally I think

Reply:
I'll remove that, THX

Reply:
Fine to me, as @philbucher says that's already solved

Reply:
done